docs/scope-v2.md

```diff
@@ -0,0 +1,158 @@
+# Zakres etapu v2
+
+Projekt rozwijany jest w modelu iteracyjnym, co sprawia, że z jednej strony jest określony szeroki zakres możliwego rozwoju projektu. Z drugiej jednak strony są określone funkcjonalności, których realizacja jest możliwa bezpośrednio lub które powinny być zrealizowane w pierwszej kolejności.
+
+Niniejszy dokument ma na celu zdefiniowania minimalnego zakresu funkcjonalności, który jest konieczny, aby uznać za zakończoną realizacje etapu `v2` aplikacji.
+
+Celem etapu v2 jest zmiana technologii przy minimalnym rozszerzeniu zakresu funkcjonalności aplikacji. Zakres funkcjonalności uległ nieznacznemu poszerzeniu ze względu na ich duże znaczenie dla organizacji lub weryfikacje architektoniczną.
+
+## Zakres zadań
+
+W ramach etapu konieczne jest:
+
+* projektowanie i implementacja
+* wdrożenie w środowisku organizacji:, w tym:
+  * migracja danych z etapu `v1`
+  * nieznaczne korekty funkcjonalności w celu zagwarantowania efektywnego wykorzystania
+* całkowite wygaszenie rozwiązania z etapu `v1`
+
+Etap wdrożenia dopuszcza wprowadzenie niedostępności systemu, nawet na okres tygodnia dni.
+
+## Architektura
+
+Aplikacja powinna uwzględniać:
+
+* serwer API – wykonany w Django udostępniający API w technologii REST
+* interfejs użytkownika – wykonany w React wykorzystujący serwer API
+* serwer plików zgodny z S3 – przyjmujący, przechowujący i udostępniający dane binarne
+
+Dla systemu przyjęto następujące wymagania dostępności:
+
+* RPO – (TODO: Ustalić z Zarządem)
```

Comment:
@mrbojko, uzgodnisz adekwatne informacje?

Reply:
Ustaliłem, że RPO w innych projektach jak dotychczas jest jak najbardziej OK

Reply:
RPO i RTO to powinna być decyzja kierownictwa. Osoby zaangażowane w projekt Stowarzyszenia nie znają standardów Stowarzyszenia.

Reply:
@ad-m powiedz proszę czego dokładnie oczekujesz. Moja odpowiedź, że RPO na obecnym poziomie jest OK powstała na podstawie rozmowy z Kasią Batko. Czy mam zwrócić się do Zarządu o stworzenie jakiegoś dokumentu regulującego RPO i RTO dla small_eod i wrzucić go na githuba?

Reply:
Ja potrzebuje absolutnej wartości RPO i RTO, którą uzgodniłeś z Kasią. Akceptuje niedostępność minuty, godziny, dni w sytuacji krytycznej? Odesłanie do poprzedniej wartości nie dostarcza odpowiedz, bo taki sposób niczego nie mówi osobom, które rozwijają projekt.

Reply:
https://github.com/watchdogpolska/infra/blob/9104ac2a99f3899302881148d78ec920ebafd9ba/docs/backup/document.md

Reply:
Dziękuje, nawet nie pamiętałem, że było to tam wpisane.